lib/sparkpost_rails/delivery_method.rb

```diff
@@ -68,14 +68,15 @@ def prepare_recipients_from mail, sparkpost_data
 
     def prepare_addresses emails, names
       emails = [emails] unless emails.is_a?(Array)
-      emails.each_with_index.map {|email, index| prepare_address(email, index, names) }
+      header_to = emails.join(",")
+      emails.each_with_index.map {|email, index| prepare_address(email, index, names, header_to) }
```

Comment:
Simply join all email addresses with "," and pass into prepare_address. I am not sure if we should also be combing the names here as well?  For example, should the header_to show email+name combinations such as heh@heh.com <Mr. Heh>